src/cast/Cast.js

```diff
@@ -95,26 +95,27 @@ export class Cast {
 
     /**
      * @param {TPermissive} x
+     * @param {string} dataPath
      * @returns {UplcData}
      */
-    toUplcData(x) {
-        return schemaToUplc(this.schema, x)
+    toUplcData(x, dataPath = "") {
```

Comment:
I'm OK with being able to pass in `dataPath` as an argument here, but could you add a note about the use-case?

Reply:
yup, will do